components/ledger/ledger-utxo-flow/src/main/kotlin/net/corda/ledger/utxo/flow/impl/transaction/UtxoTransactionBuilderImpl.kt

```diff
@@ -83,6 +87,9 @@ data class UtxoTransactionBuilderImpl(
     }
 
     override fun addOutputState(contractState: ContractState, encumbrance: Int?): UtxoTransactionBuilder {
+        if (notary == null) {
+            throw(CordaRuntimeException("Adding Output states is not possible until the notary is not set!"))
```

Comment:
The double negative here is a bit confusing.
```suggestion
            throw(CordaRuntimeException("Adding Output states is not possible until the notary has been set!"))
```

Reply:
Thanks. I have updated these.